.github/workflows/build.yaml

```diff
@@ -19,6 +17,10 @@ jobs:
           - aws
           - ibmcloud
           - libvirt
+        runner:
+          - ubuntu-latest
```

Comment:
Just a thought, but I wonder if hooking this into the `runs-on` field e.g. `runs-on: ${{ matrix.runner }}`, might be more intuitive if we decided to add support for more OSs in future, and would remove a duplication of `ubuntu-latest`

Reply:
It does make sense. I will change that.